src/runners/signer_runner.ts

```diff
@@ -0,0 +1,47 @@
+import bodyParser = require('body-parser');
+import * as cors from 'cors';
+import * as express from 'express';
+import * as asyncHandler from 'express-async-handler';
+
+import { META_TRANSACTION_PATH } from '../constants';
+import { SignerHandlers } from '../handlers/signer_handlers';
+import { logger } from '../logger';
+import { errorHandler } from '../middleware/error_handling';
+import { requestLogger } from '../middleware/request_logger';
+import { SignerService } from '../services/signer_service';
+
+if (require.main === module) {
+    (async () => {
+        const app = express();
+        app.use(requestLogger());
+        app.use(cors());
+        app.use(bodyParser.json());
+
+        const signerService = new SignerService();
+        const handlers = new SignerHandlers(signerService);
+        /**
+         * POST Transaction endpoint takes a signed 0x Transaction and sends it to Ethereum
+         * for execution via `executeTransaction`.
+         *
+         * https://0x.org/docs/guides/v3-specification#executing-a-transaction
+         */
+        app.post(
+            `${META_TRANSACTION_PATH}/fill`,
+            asyncHandler(handlers.submitZeroExTransactionIfWhitelistedAsync.bind(handlers)),
+        );
+
```

Comment:
Should it be called `/submit`? Technically we could use this for any `ZeroExTransaction` in the future right?